crm_stage_probability/tests/test_crm_lead_probability.py

```diff
@@ -0,0 +1,95 @@
+# Copyright 2020 Camptocamp SA
+# License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl)
+from odoo.exceptions import UserError
+from odoo.tests import TransactionCase
+
+
+class TestCrmLeadProbability(TransactionCase):
+    @classmethod
+    def setUpClass(cls):
+        super().setUpClass()
+        cls.stage_new = cls.env.ref("crm.stage_lead1")
+        cls.stage_qualified = cls.env.ref("crm.stage_lead2")
+        cls.stage_proposition = cls.env.ref("crm.stage_lead3")
+        cls.stage_won = cls.env.ref("crm.stage_lead4")
+        cls.opportunity_1 = cls.env.ref("crm.crm_case_32")
+
+    def test_update_probability(self):
+        self.assertEqual(self.opportunity_1.stage_id, self.stage_qualified)
+        self.assertFalse(self.opportunity_1.is_automated_probability)
+        self.assertFalse(self.opportunity_1.is_stage_probability)
+        self.opportunity_1.write({"stage_id": self.stage_new.id})
```

Comment:
Hi, @vib-adhoc !
I have reviewed the codecov messages and maybe an oportunity with a null stage_id could make test entering in some missing lines.